documentation/modules/exploit/linux/http/linear_emerge_unauth_rce_cve_2019_7256.md

```diff
@@ -0,0 +1,188 @@
+## Vulnerable Application
+
+Nortek Security & Control, LLC (NSC) is a leader in wireless security, home automation and personal safety systems and devices.
+The eMerge E3-Series is part of Linear’s access control platform, that delivers entry-level access control to buildings.
+It is a web based application where the HTTP web interface is typically exposed to the public internet.
+
+The Linear eMerge E3 versions `1.00-06` and below are vulnerable to an unauthenticated command injection remote root exploit
+that leverages card_scan_decoder.php.
+This can be exploited to inject and execute arbitrary shell commands as the root user through the `No` and `door` HTTP GET parameter.
+A successful exploit could allow the attacker to execute arbitrary commands on the underlying operating system with the root privileges.
+
+Building Automation and Access Control systems are at the heart of many critical infrastructures, and their security is vital.
+Executing attacks on these systems may enable unauthenticated attackers to access and manipulate doors, elevators, air-conditioning systems,
+cameras, boilers, lights, safety alarm systems in an entire building.
+
+This issue affects all Linear eMerge E3 versions up to and including `1.00-06`.
+
+Installing a vulnerable test bed requires a Linear eMerge E3 access controller with the vulnerable software loaded.
```

Comment:
I doubt we have one of these.  Can you provide us with a packet capture demonstrating the exploit?  Please be sure to remove any sensitive data from it and email msfdev[at]metasploit.com.
Thanks!

Reply:
pcap files have been shared !